reducer-todo-new/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React, { useReducer } from 'react';
+import {reducer, initialState} from "./reducers/reducer";
+import './to_do_list.scss'
+// import 'semantic-ui-css/semantic.min/css'
+
+import ToDoCard from './components/ToDoCard'
+
+function App() {
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  return (
+    <ToDoCard state = {state} dispatch = {dispatch} />
```

Comment:
Your app should be functioning, but the name ToDoCard sounds more like a single item than a whole app.

Reply:
I might have just put all the components inside ToDoCard inside here and just omitted that layer of abstraction all together